server.js

```diff
@@ -137,7 +137,13 @@ function initialisePublic (app) {
   app.use('/javascripts', express.static(path.join(__dirname, '/public/assets/javascripts'), publicCaching))
   app.use('/images', express.static(path.join(__dirname, '/public/images'), publicCaching))
   app.use('/stylesheets', express.static(path.join(__dirname, '/public/assets/stylesheets'), publicCaching))
-  app.use('/', express.static(path.join(__dirname, '/node_modules/govuk-frontend/govuk/')))
+
+  if (process.env.NGINX_CACHING_ENABLED === 'true') {
+    app.use('/', express.static(path.join(__dirname, '/node_modules/govuk-frontend/govuk/'), publicCaching))
```

Comment:
I don't think setting this header requires being behind an NGINX proxy, it should be OK to set this uniformly as with the other resources.

This could be naturally reviewed if and when the card payments page is separated from the product pages.

Reply:
The Hufflepuff team will be putting in the NGINX layer so that new requests for static assets will not hit Node - but served from NGINX instead.

Good point about reviewing our caching strategy once we seperate product pages.